src/SFML/System/Android/ResourceStream.hpp

```diff
@@ -98,10 +92,18 @@ class SFML_SYSTEM_API ResourceStream : public InputStream
     std::int64_t getSize() override;
 
 private:
+    ////////////////////////////////////////////////////////////
+    // Types
+    ////////////////////////////////////////////////////////////
+    struct AAssetDeleter
```

Comment:
Not entirely sure but I guess exporting this is needed for compiler-generated destructor to find this when ResourceStream is used in other modules (e.g. Graphics module).
```suggestion
    struct SFML_SYSTEM_API AAssetDeleter
```

Reply:
You can thank MinGW for that. No other compiler demanded we export this symbol. I know we've had to do this once before for another private type but I forget where.

Reply:
Nevermind, the problem we had in a similar case in `InputSoundFile` (see b48f4b70a7a9781af9b69c6f8a4382a45594999f) was only with MinGW. I don't think it will be problem for this Android-specific code.